default-fields/0000-default-fields.md

```diff
@@ -6,31 +6,206 @@
 # Summary
 [summary]: #summary
 
-One para explanation of the feature.
+Allow struct definitions to supply default values for individual fields,
+and then allow those fields to be omitted from struct initialisation:
+
+```rust
+struct Foo {
+    a: &'static str,
+    b: bool = true,
+    c: i32,
+    d: Vec<i32> = Vec::default()
```

Comment:
hmm, this feels a bit like overkill. I was thinking more that the `Default` trait could be a lang item. However, all this talk of syntactic sugar for something that looks like a function call makes me nervous. I saw `..` suggested as special syntax for this at some point. However, I'm thinking that is might be best to drop the default special case to start with and just list it as an alternative.